### PR TITLE
newrelic-infrastructure-v3: restrict interval to [10, 45]s

### PR DESCRIPTION
**Pull request by roobre**
#### What this PR does / why we need it:

Causes the chart to fail to render if `common.config.interval` is not in the `[10, 45]s` range.

This check can be overridden by setting the undocumented and unsupported flag `forceUnsupportedInterval` to `true`.

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Chart Version bumped
- [ ] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[mychartname]`)

### CODE REVIEW

**Review by paologallinaharbur — APPROVED** [review]
LGTM

File: charts/newrelic-infrastructure-v3/tests/interval_test.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+      - failedTemplate:
+          errorMessage: "raw: Intervals smaller than 10s are not supported"
+  - it: Does not fail with override
+    set:
+      licenseKey: test
+      cluster: test
+      common:
+        config:
+          interval: 1s
+      forceUnsupportedInterval: true
```
Comment (paologallinaharbur) [inline charts/newrelic-infrastructure-v3/tests/interval_test.yaml:32]: just a doupt, is it asserting/doing anything? this test?

Comment (paologallinaharbur) [reply charts/newrelic-infrastructure-v3/tests/interval_test.yaml:32]: maybe it is better just to check the interval was correctly set

Comment (roobre) [reply charts/newrelic-infrastructure-v3/tests/interval_test.yaml:32]: Right now it's checking that the chart rendering does not fail, but that's a good point: I can check for the interval as well!

File: charts/newrelic-infrastructure-v3/tests/interval_override_test.yaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+      licenseKey: test
+      cluster: test
+      common:
+        config:
+          interval: 1s
+      forceUnsupportedInterval: true
+    asserts:
+      - matchRegex:
+          path: data.nri-kubernetes\.yml
+          pattern: 'interval: 1s'
```
Comment (paologallinaharbur) [inline charts/newrelic-infrastructure-v3/tests/interval_override_test.yaml:18]: nice! better that adding the whole config!

**Review by paologallinaharbur — APPROVED** [review]
thanks for addressing the corner case!

**Comment by paologallinaharbur** [comment]
you have two PR pointing to the same version
